src/frontend/Sider/Behandling/Vilkårvurdering/MeldingHvisLagringFeilet.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+import { ErrorMessage } from '@navikt/ds-react';
+
+import { useVilkår } from '../../../context/VilkårContext';
+
+export function MeldingHvisLagringFeilet(props: { vilkårId: string }) {
+    const { feilmeldinger } = useVilkår();
+
+    const feilmelding = feilmeldinger[props.vilkårId];
+
+    return (
+        <>
+            {feilmelding && (
+                <ErrorMessage size={'small'}>
+                    Oppdatering av vilkår feilet: {feilmelding}
+                </ErrorMessage>
+            )}
+        </>
+    );
+}
```

Comment:
Burde vi kanskje bruke `default export` på disse type komponentene? Der det kun finnes en komponent i en fil som eksponeres? 
Husk å oppdatere import i `EndreDelvilkår`

```suggestion
const MeldingHvisLagringFeilet(props: { vilkårId: string }) {
    const { feilmeldinger } = useVilkår();

    const feilmelding = feilmeldinger[props.vilkårId];

    return (
        <>
            {feilmelding && (
                <ErrorMessage size={'small'}>
                    Oppdatering av vilkår feilet: {feilmelding}
                </ErrorMessage>
            )}
        </>
    );
}

export default MeldingHvisLagringFeilet
```